sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample1_RecognizeContentFromFile.cs

```diff
@@ -21,49 +21,54 @@ public async Task RecognizeContentFromFile()
 
             FormRecognizerClient client = new FormRecognizerClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
 
-            string invoiceFilePath = FormRecognizerTestEnvironment.CreatePath("Invoice_1.pdf");
+            string filePath = FormRecognizerTestEnvironment.CreatePath("Invoice_1.pdf");
 
-            using (FileStream stream = new FileStream(invoiceFilePath, FileMode.Open))
+            #region Snippet:FormRecognizerRecognizeFormContentFromFile
+            //@@ string filePath = "filePath";
+            using var stream = new FileStream(filePath, FileMode.Open);
+
+            Response<FormPageCollection> response = await client.StartRecognizeContentAsync(stream).WaitForCompletionAsync();
+            FormPageCollection formPages = response.Value;
+
+            foreach (FormPage page in formPages)
             {
-                FormPageCollection formPages = await client.StartRecognizeContentAsync(stream).WaitForCompletionAsync();
-                foreach (FormPage page in formPages)
-                {
-                    Console.WriteLine($"Form Page {page.PageNumber} has {page.Lines.Count} lines.");
+                Console.WriteLine($"Form Page {page.PageNumber} has {page.Lines.Count} lines.");
 
-                    for (int i = 0; i < page.Lines.Count; i++)
-                    {
-                        FormLine line = page.Lines[i];
-                        Console.WriteLine($"    Line {i} has {line.Words.Count} word{(line.Words.Count > 1 ? "s" : "")}, and text: '{line.Text}'.");
+                for (int i = 0; i < page.Lines.Count; i++)
+                {
+                    FormLine line = page.Lines[i];
+                    Console.WriteLine($"  Line {i} has {line.Words.Count} word{(line.Words.Count > 1 ? "s" : "")}, and text: '{line.Text}'.");
```

Comment:
This is a minor detail, but do you think it would make it clearer if it was:
```cs
...{(line.Words.Count > 1 ? "words" : "word")}...
```
instead? I think it makes the possibilities more evident.

Reply:
By the way, I believe `line.Words.Count == 1 ? "word" : "words"` would be preferred, since it includes the case where there are no lines.